R/do_findChromPeaks-functions.R

```diff
@@ -3668,3 +3711,49 @@ peaksWithCentWave <- function(int, rt,
     }
     lm
 }
+
+
+#' @description
+#'
+#' Calculate beta parameters for a chromatographic peak, both its similarity
+#' to a bell curve of varying degrees of skew and the standard deviation of the
+#' residuals after the best-fit bell is normalized and subtracted.
+#'
+#' @param eic_ints A numeric vector corresponding to the peak intensities
+#' @param skews A numeric vector of the skews to try, corresponding to the
+#' shape1 of dbeta with a shape2 of 5. Values less than 5 will be increasingly
+#' right-skewed, while values greater than 5 will be left-skewed.
+#'
+#' @author William Kumler
+#'
+#' @noRd
+.get_beta_values <- function(eic_ints, skews=c(3, 3.5, 4, 4.5, 5)){
+  if(length(eic_ints)<5){
```

Comment:
I would suggest to replace 0 intensities with `NA` here to avoid using them in the `cor` call:

```
eic_ints[which(eic_ints == 0)] <- NA_real_
if (sum(!is.na(eic_ints)) < 5) {
    best_cor <- NA_real_
    best_snr <- NA_real_
} else ...
```

Reply:
Ignore that - better to check my last comment.